BridgeSDK/Networking/SBBNetworkManager.m

```diff
@@ -251,6 +251,12 @@ - (NSURLSession *)backgroundSession
       dispatch_once(&onceToken, ^{
         NSURLSessionConfiguration *config = [NSURLSessionConfiguration backgroundSessionConfigurationWithIdentifier:kBackgroundSessionIdentifier];
         bgSession = [NSURLSession sessionWithConfiguration:config delegate:self delegateQueue:nil];
+       
+        // Set the configuration to discretionary 
+        [config setDiscretionary: YES]; 
```

Comment:
I wonder if this should be set before the `bgSession` is constructed? Although if you've tested this and it works, then it doesn't matter.

Reply:
Yea I'll make that change - I haven't verified it yet because I was waiting until we set up a web hook in Slack to easily monitor transfer/upload activity. Should be able to have that set up so this can be verified this week